src/main/java/com/epam/test_generator/controllers/CaseController.java

```diff
@@ -117,8 +118,27 @@
         method = RequestMethod.PUT, consumes = "application/json")
     public ResponseEntity<Void> updateCase(@PathVariable("suitId") long suitId,
                                            @PathVariable("caseId") long caseId,
-                                           @RequestBody @Valid EditCaseDTO editCaseDTO) {
-        caseService.updateCase(suitId, caseId, editCaseDTO);
```

Comment:
Why you changed EditCaseDTO to CaseDTO? That means that we can pass a nested list of steps again?

Reply:
Fixed, replaced CaseDTO with EditCaseDTO in update methods (for single Case and for list)